java/code/src/com/redhat/rhn/frontend/strings/jsp/StringResource_en_US.xml

```diff
@@ -6314,8 +6314,8 @@ organization. You may grant or remove the organization administrator role in the
       <trans-unit id="pkg.lock.summary" xml:space="preserve">
         <source>
                 You can lock packages so they will remain unmodified.
-                However, it is important to keep in mind that requesting a 
-                lock might be delayed. If a package has been updated 
+                However, it is important to keep in mind that requesting a
+                lock might be delayed. If a package has been updated
```

Comment:
I hope the trailing whitespaces are really not needed. If possible check the text in the UI

Reply:
I reverted this change. It's not necessary.